### ISSUE TITLE
I wish /docs/user-guide/kubectl-cheatsheet/ fixed the code sample for kubectl get nodes -o jsonpath='{.items[*].status.addresses[?(@.type == "ExternalIP")].address}'

### ISSUE THREAD

**Issue by azell** [kubernetes/kubernetes.github.io#701]
I wish /docs/user-guide/kubectl-cheatsheet/ fixed the code sample for

```
kubectl get nodes -o jsonpath='{.items[*].status.addresses[?(@.type == "ExternalIP")].address}'
```

**Comment by runseb** [comment]
@azell what is wrong with that command ? it seems to work fine for me.

**Comment by azell** [comment]
```
$ KUBECONFIG=config kubectl version                                                                           
Client Version: version.Info{Major:"1", Minor:"2", GitVersion:"v1.2.4", GitCommit:"3eed1e3be6848b877ff80a93da3785d9034d0a4f", GitTreeState:"clean"}
Server Version: version.Info{Major:"1", Minor:"2", GitVersion:"v1.2.4", GitCommit:"3eed1e3be6848b877ff80a93da3785d9034d0a4f", GitTreeState:"clean"}

$ KUBECONFIG=config kubectl get nodes -o jsonpath='{.items[*].status.addresses[?(@.type=ExternalIP)].address}'     
Error executing template: unrecognized identifier ExternalIP. Printing more information for debugging the template:
	template was:
		{.items[*].status.addresses[?(@.type=ExternalIP)].address}
	object given to jsonpath engine was:
...
error: error executing jsonpath "{.items[*].status.addresses[?(@.type=ExternalIP)].address}": unrecognized identifier ExternalIP
```

**Comment by runseb** [comment]
you have `@.type=ExternalIP)` instead of `@.type=="ExternalIP")`

double equals and double quotes...

**Comment by azell** [comment]
Yes, that is what I am trying to correct on the web page :).  Current URL contents:

```
# Get ExternalIPs of all nodes
$ kubectl get nodes -o jsonpath='{.items[*].status.addresses[?(@.type=ExternalIP)].address}'
```

**Comment by runseb** [comment]
Ok gotcha, I fixed it in #763
